src/actions/CreateWalletActions.js

```diff
@@ -45,15 +50,124 @@ export const createCurrencyWallet = (
     keyOptions: format ? { format } : {}
   })
     .then(edgeWallet => {
-      Actions.popTo(Constants.WALLET_LIST_SCENE)
+      if (popScene) Actions.popTo(Constants.WALLET_LIST_SCENE)
       dispatch({ type: 'UI/WALLETS/CREATE_WALLET_SUCCESS' })
       if (selectWallet) {
         dispatch(selectWalletAction(edgeWallet.id, edgeWallet.currencyInfo.currencyCode))
       }
+      return edgeWallet
     })
     .catch(async error => {
       await showModal(errorModal(s.strings.create_wallet_failed, error))
       Actions.popTo(Constants.WALLET_LIST_SCENE)
       dispatch({ type: 'UI/WALLETS/CREATE_WALLET_FAILURE' })
     })
 }
+
+export const fetchAccountActivationInfo = (currencyCode: string) => async (dispatch: Dispatch, getState: GetState) => {
+  const state = getState()
+  const account = CORE_SELECTORS.getAccount(state)
+  const currencyPluginName = Constants.CURRENCY_PLUGIN_NAMES[currencyCode]
+  const currencyPlugin = account.currencyConfig[currencyPluginName]
+  try {
+    // $FlowFixMe
+    const supportedCurrencies = currencyPlugin.otherMethods.getActivationSupportedCurrencies()
```

Comment:
I thought we had said flow fix me's killed flow for the rest of the page. What were the errors you were getting, and can we figure out how to fix.

Reply:
The `otherMethods` was not officially part of the API when I wrote this. Let me check with William to see if it's officially in the core / plugins now.